tslint.json

```diff
@@ -68,6 +68,6 @@
     },
     "rulesDirectory": [],
     "linterOptions": {
-      "exclude": ["node_modules/**/*", "**/*.js", "dev-tools/*", ".*-dist/**/*"]
+      "exclude": ["node_modules/**/*", "**/*.js", "dev-tools/global*", ".*-dist/**/*"]
```

Comment:
Why isn't this just `"node_modules/"`?

Reply:
`**/*.js` is also a pretty inefficient glob... why would tslint bother with JS files at all?

Reply:
> Why isn't this just `"node_modules/"`?

Done.

> **/*.js is also a pretty inefficient glob... why would tslint bother with JS files at all?

Good question. I've removed `jsRules`, but even without that it still checks JS! Maybe has something to do with our `checkJs`/`allowJs` settings?. Seems borked, but if I remove the JS glob from exclude i get zillions of complaints about our JS files.